.github/workflows/ci.yml

```diff
@@ -9,13 +9,29 @@ on:
       - main
 
 jobs:
+  copywrite:
+    name: Run Header Copyright
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+
+      - name: Install Copywrite
+        id: install
+        uses: hashicorp/setup-copywrite@v1.0.0
+
+      - name: Output Installed Copywrite Version
+        run: echo "Installed Copywrite CLI ${{steps.install.outputs.version}}"
+
+      - name: Run Copywrite Header Compliance
+        run: copywrite headers --plan
   test:
     runs-on: ubuntu-latest
     timeout-minutes: 5
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
```

Comment:
Just curious: why pin these by SHA? Seems like we'd want to keep semantic version compatibility guarantees but get bug fixes

Reply:
I was following an example from someone in the copywrite Slack channel. I'd be happy to go back through and update the other repos to use a semantic version rather than a SHA.